### ISSUE TITLE
Trouble upgrading from 1.1.3 to 1.2.0-beta.8

### ISSUE THREAD

**Issue by yreynhout** [SQLStreamStore/SQLStreamStore#377]
You have an app A that depends on component B and component B depends on `>= 1.1.3` of SqlStreamStore. Now, app A itself depends on `1.2.0-beta.8` of SqlStreamStore. Build and publish work as expected, that is the dependency becomes `1.2.0-beta.8`.`1.1.3` has as its `AssemblyVersion` ... `1.1.3.0`.

`MinVer` was introduced as part of `1.2.0` development. Due to how `MinVer` hands out the `AssemblyVersion` (as per its documentation https://github.com/adamralph/minver#version-numbers) it has set it to `1.0.0.0`.

This causes a `FileLoadException` at runtime when app A is started and invokes types in component B that use SqlStreamStore, stating that it's looking for `1.1.3.0` which it can't find, because, well, there's only `1.0.0.0`.

```
Could not load file or assembly 'SqlStreamStore, Version=1.1.3.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (0x80131040))\n ---> System.IO.FileLoadException: Could not load file or assembly 'SqlStreamStore, Version=1.1.3.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)\nFile name: 'SqlStreamStore, Version=1.1.3.0, Culture=neutral, PublicKeyToken=null
```

@adamralph @damianh

**Comment by yreynhout** [comment]
The resolution _might_ be to take over the assembly version assignment from `MinVer` and set it to `1.2.0.0` once it is released and keeping it for the entire lifecycle of `1.2+`

**Comment by adamralph** [comment]
It sounds like you should keep the old versioning scheme for `AssemblyVersion` for the 1.x range:

```xml
<Target Name="MyTarget" AfterTargets="MinVer" Condition="'$(MinVerMajor)' == '1'">
  <PropertyGroup>
    <AssemblyVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).0</AssemblyVersion>
  </PropertyGroup>
</Target>
```

**Comment by yreynhout** [comment]
Addressed by #378
